profile/CSIP.xml

```diff
@@ -1382,7 +1384,7 @@
             </requirement>
         </content_files>
         <behavior_files>
-            <requirement>
+            <requirement ID="REF_METS_">
```

Comment:
This one has no ID number?

Reply:
We dont use the technical requirements part so far...... Removed it.